spec/models/mixins/ci_feature_mixin_spec.rb

```diff
@@ -19,23 +19,10 @@
       expect(service.service_resources.first.resource.retireable?).to eq(false)
     end
 
-    context "service" do
-      context "with type" do
-        let(:service1) { FactoryBot.create(:service_ansible_tower, :type => ServiceAnsibleTower) }
-        it "is retireable" do
-          FactoryBot.create(:service_resource, :service => service, :resource => service1)
+    it "service is retireable" do
+      FactoryBot.create(:service_resource, :service => service, :resource => FactoryBot.create(:service_ansible_tower, :type => ServiceAnsibleTower))
 
-          expect(service.service_resources.first.resource.retireable?).to eq(true)
-        end
-      end
-
-      context "without type" do
```

Comment:
Doesn't this still need to be tested?

Reply:
It's here: https://github.com/ManageIQ/manageiq/pull/18283/files#diff-92a7338185f2ababc762a33994ba1eebR687